src/shogun/classifier/AveragedPerceptron.cpp

```diff
@@ -67,9 +67,12 @@ bool CAveragedPerceptron::train_machine(CFeatures* data)
 	while (!(CSignal::cancel_computations()) && (!converged && iter<max_iter))
 	{
 		converged=true;
+		SG_INFO("Iteration Number : %d\n", iter);
```

Comment:
Please also print the maximum iteration number.
`SG_INFO("Iteration number : %d of max %d\n", iter, max_iter);`